bin/nrnivmodl_makefile_cmake.in

```diff
@@ -67,10 +67,10 @@ C_GREEN := \033[32m
 
 # Take the main and link with nrnmech.
 # RPATH is set for DESTDIR_RPATH and coreneuron lib
-$(special): $(mech_lib)
-	@printf " => $(C_GREEN)LINKING$(C_RESET) executable $(special)\n"
-	$(CXX_LINK_EXE) -I $(incdir) $(datadir)/coreneuron.cpp -o $(special) \
```

Comment:
yeah, I suspected that this was never used in NEURON and just copied from CoreNEURON!

Reply:
Indeed. Most of the logic was valid however.